src/views/blocks/tickets/item-inactive.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Block: Tickets
+ * Inactive Ticket Item
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/blocks/tickets/item-inactive.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version 4.9
+ *
+ */
+
+$sale_past = $this->get( 'sale_past' );
+$classes   = array(
+	'tribe-block__tickets__item',
+	'tribe-block__tickets__item--inactive',
+);
+
+$context = array(
+	'sale_past' => $sale_past,
+);
+?>
+<div class="<?php echo implode( ' ', get_post_class( $classes ) ); ?>">
```

Comment:
Just noticed this, it needs to have esc_attr()

Reply:
Can you also add esc_attr() anywhere else you see implode() happening in these block views? I see it in src/views/blocks/tickets/item.php too